packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -226,11 +265,14 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
   &.mdc-textfield--disabled {
     border-bottom: none;
     background-color: rgba(black, .02);
-    color: rgba(black, .38);
+
+    @include mdc-theme-prop(background-color, text-textfield-background-on-light);
+    @include mdc-theme-prop(color, text-textfield-placeholder-on-light);
 
     @include mdc-theme-dark(".mdc-textfield", true) {
       border-bottom: none;
-      background-color: rgba(white, .05);
+
+      @include mdc-theme-prop(background-color, text-textfield-background-on-dark);
```

Comment:
@include mdc-theme-prop(color, text-textfield-placeholder-on-dark)?? Should that be included as well?